components/example-notebook-servers/README.md

```diff
@@ -0,0 +1,139 @@
+# Example Notebook Servers
+
+**DISCLAIMER:** The Notebooks Working Group provides these Dockerfiles and
+their images as examples only, they are not tested with our CI/CD pipelines
+and are not certified to work in every situation. As such, issues related to
+these images will be dealt with in a best efforts approach. The Notebooks
+Working Group will work on officially supporting some of these Notebook
+Server images in the near future. If you do encounter a problem in one of
+these images, contributions and issue reports are greatly appreciated.
+
+## Introduction
+
+This directory contains Dockerfiles that are intended to show users how to
+create images for various types of Notebook Servers that can run on Kubeflow.
+There is a common base images that starts from Ubuntu 20.04, adds some
+dependencies and common utilities from APT, installs `kubectl`, creates the
+user account for the downstream images and adds the
+[S6-overlay](https://github.com/just-containers/s6-overlay) for the init system.
+From this base image, images for Jupyter, RStudio and Visual Studio Code
+(through [code-server]([https://github.com/cdr/code-server)) are created.
+From the Jupyter image, images are created that show users how to install
+TensorFlow and PyTorch both with and without support for NVIDIA GPUs.
+In turn, these images are extended with an opinionated set of common tools
+and packages to help new users to quickly get started with Kubeflow.
+
+**Lineage of the provided example Notebook Server Images:**
+
+![Flow Chart of example Notebook Servers](./kubeflow-example-notebook-servers.png)
+
+## Creating custom Notebook Server images
+
+Because the needs of users varies greatly, the provided images were created to
+be easily extended by using the [S6-overlay](https://github.com/just-containers/s6-overlay)
+init system and including [Conda](https://github.com/conda-forge/miniforge).
+
+### Installing packages
+
+The most common customization to Notebook Server images will be to install
+a selection of packages specific that will be needed in the Notebook Server.
+Users can install packages in a running Notebook Server using `pip install ...`
+or `conda install ...`. However, packages installed in this manner will not
+persist if the Notebook Server is stopped or the Pod is restarted.
+While it is possible to install Python packages in the user's home directory
+by using `pip install --user ...`, this can cause unexpected behaviour if the
+workspace volume is later used in a different Notebook Server, as the packages
+present in the home directory might be used rather than the ones present in the
+Notebook Server's image. For this reason it is recommended to create custom
```

Comment:
I am not sure the discussion about the lifecycle of packages and PVCs is relevant to this document. I would remove it

Reply:
While not directly relevant to this document, this very situation has been the cause of issues in the past so I do think this should be stated somewhere. I agree, it is a bit out of place here. I'll leave it in here for the moment until while I think of what to do with it. If needed, it can quickly be removed before merging. 

I'm thinking a `Best practices and considerations` section might be on option (in this README, as well as on the website, which I'm guessing will be mostly copy/paste from here with some formatting changes).

Reply:
While I think this wording can be simplified, we should definitely discuss the implications of how kubeflow mounts PVCs to the home directory for `pip install`